src/tests/ui/planner/sidepanel.ts

```diff
@@ -1,4 +1,4 @@
-import { $, ElementFinder, by} from 'protractor';
+import { $, ElementFinder, by, element} from 'protractor';
```

Comment:
Do we need `element`?

Reply:
remove this is creating merge conflict in your branch